portfolio/components/ui/Figures/BetweenReceiptVisualization/BetweenReceiptVisualization.module.css

```
@@ -0,0 +1,431 @@
+  justify-content: center;
+}
+
+.fadeOut {
+  animation: fadeOut 0.4s ease-out forwards;
+}
+
+.fadeIn {
+  animation: fadeIn 0.4s ease-in forwards;
+}
+
+@keyframes fadeOut {
```

Stylelint (17.2.0)

[error] Expected keyframe name "fadeOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)